source/integrations/setup-nginx-proxy-with-minio.rst

```diff
@@ -0,0 +1,193 @@
+======================================
+Configure Nginx Proxy for MinIO Server
+======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following documentation covers the minimum settings required to configure Nginx to proxy requests to MinIO.
+
+This documentation assumes the following:
+
+- An existing `Nginx <http://nginx.org/en/download.html>`__ deployment
+- An existing :ref:`MinIO <minio-installation>` deployment
+- A DNS hostname which uniquely identifies the MinIO deployment
+
+There are two models for proxying requests to the MinIO Server API and the MinIO Console:
+
```

Comment:
Choosing to not deal with :443 or https here. Not sure if that makes a difference.